vello/src/scene.rs

```diff
@@ -334,23 +345,284 @@ impl<'a> DrawGlyphs<'a> {
     ///
     /// The `style` parameter accepts either `Fill` or `&Stroke` types.
     pub fn draw(mut self, style: impl Into<StyleRef<'a>>, glyphs: impl Iterator<Item = Glyph>) {
-        let resources = &mut self.encoding.resources;
-        self.run.style = style.into().to_owned();
-        resources.glyphs.extend(glyphs);
-        self.run.glyphs.end = resources.glyphs.len();
-        if self.run.glyphs.is_empty() {
+        let font_index = self.run.font.index;
+        let font = skrifa::FontRef::from_index(self.run.font.data.as_ref(), font_index).unwrap();
+        if font.colr().is_ok() {
```

Comment:
What's the actual right way to detect this?

Reply:
This is fine but I’d also check for cpal here since you need both.